api/src/main/java/org/pmiops/workbench/db/model/DbStorageEnums.java

```diff
@@ -321,25 +316,6 @@ public static Domain domainIdToDomain(String domainId) {
     return DOMAIN_ID_MAP.inverse().get(domainId);
   }
 
-  // Education
```

Comment:
All of these are duplicated in DemographicSurveyEnum, often with different values.  Yikes!  These were not used, so let's kill them before they escape.

*** note: we have not done a definitive check as to whether these alternative values were used anywhere

Reply:
Update: these were apparently never used